galsim/wfirst/wfirst_bandpass.py

```diff
@@ -65,7 +66,10 @@ def getBandpasses(AB_zeropoint=True, exptime=None, thin_err=1.e-4):
                             True]
     @param exptime          Exposure time to use for setting the zeropoint; if None, use the default
                             WFIRST exposure time, taken from galsim.wfirst.exptime.  [default: None]
-    @param thin_err         Relative error allowed when thinning the bandpasses.  [default: 1e-4]
```

Comment:
The parameter `default_thin_trunc` should be listed explicitly here.  And I'd suggest adding after that
```
@params **kwargs           Other kwargs are passed to either `bandpass.thin()` or `bandpass.truncate()` as appropriate.
```
Also, I think some of the below would be slightly simpler if the `default_thin_trunc` kwarg was not part of the `kwargs` dict.  So I'd put it explicitly in the function signature before `**kwargs`.